<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RangeWidget</class>
 <widget class="QWidget" name="RangeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>129</width>
    <height>43</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSpinBox" name="lowerBound"/>
   </item>
   <item>
    <widget class="QSpinBox" name="upperBound"/>
   </item>
   <item>
    <widget class="QPushButton" name="closeButton">
     <property name="minimumSize">
      <size>
       <width>21</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>21</width>
       <height>21</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../resource.qrc">
       <normaloff>:/resource/icons/icon_action_cross.png</normaloff>:/resource/icons/icon_action_cross.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
